### DISCUSSION TITLE
Repo Defaults like privat=true and branch=main

### DISCUSSION THREAD

**Discussion by bw9ubwo** [charmbracelet/soft-serve#587] [Q&A] [answered]
Hi, I am trying to figure out how I can set defaults for new repositories. If I create a new one I want that the default branch is main not master and the repo is private.

**Comment by aymanbagabas** [comment, marked as answer]
Hi @bw9ubwo! In terms of default branch name, you could change that in Git config for the user running Soft Serve `git config --global init.defaultBranch main`.

You need to use the `-p` flag on the SSH command-line interface `ssh your-soft-server repo create -p repo-name` to create private repositories.
If you're the only one using the server, and don't care about multiple users, you could change the server `anon-access` to `no-access` and `allow-keyless` to `false`. Refer to the [authentication](https://github.com/charmbracelet/soft-serve#authentication) section for more information.
